operations/cani/Add_A_Cabinet_To_SLS.md

```diff
@@ -0,0 +1,27 @@
+# Add A Cabinet To SLS using CANI
+
+Begin by [initializing a session](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_init/), which will import data from SLS and create metadata in SLS' `ExtraProperties` field allowing the process to continue.
+
+```text
+ncn# cani alpha session init csm \
+  --csm-keycloak-username username \
+  --csm-keycloak-password password \
+  --csm-api-host api-gw-service-nmn.local # initialize a session by importing SLS data to cani
+```
+
+> Note: SLS data may be unstable if `cani` has never been run against it, in which case `cani` will not continue.  Resolve any of the data integrity errors before running the above command again.  Known issues and solutions can be found [here](https://cray-hpe.github.io/cani/latest/troubleshooting/known_errors/).
+
+[View available cabinet types and add one](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_add_cabinet/) using suggested values.
+
+```text
+ncn# cani alpha add cabinet -L # see supported cabinet types
+ncn# cani alpha add cabinet <cabinet type> --auto # add a cabinet using recommended values
+```
+
+[Apply](https://cray-hpe.github.io/cani/latest/commands/cani_alpha_session_apply/) the changes, writing the new data directly to SLS.
+
+```text
+ncn# cani alpha session apply # push changes to SLS
+```
+
```

Comment:
May want a pointer that the cabinet will not have any blades in it, and blades will need to be added.

Reply:
Fixed